### ISSUE TITLE
Kitti360 ground truth labels

### ISSUE THREAD

**Issue by Iana-Zhura** [hustvl/Symphonies#14]
Hello!
Thank you for your interesting work!
I have a question regarding kitti360 dataset. Did you use ground truth labels provided by SSCbench (from here https://github.com/ai4ce/SSCBench/tree/main/dataset/KITTI-360)? Did you encounter a problem that there are unlabeled areas in ground truth? Can you share please what preprocessing steps you have done with kitti360 labels before training. Thank you in advance!

**Comment by npurson** [comment]
We indeed encountered the same issue where a considerable portion of the areas were labeled as "ignored". We did not perform any preprocessing and instead trained and evaluated the model with the ignored voxels masked, as is typically done.

But for visualization, we found it necessary to replace the predictions of gt ignored voxels with the "empty" class to generate interpretable visualizations. Below, we present our visualization results on SSCBench-KITTI-360:

Prediction:
![000190_pred](https://github.com/hustvl/Symphonies/assets/58366270/8931bc9f-af62-4bba-b548-4ed78e211769)

Ground Truth:
![000190_gt](https://github.com/hustvl/Symphonies/assets/58366270/5b61856d-f4f5-44aa-8128-b10e0facac80)

I hope this helps!
